devito/operator.py

```diff
@@ -231,7 +231,7 @@ def compile(self):
         """
         if self._lib is None:
             # No need to recompile if a shared object has already been loaded.
-            return jit_compile(self.ccode, self._compiler)
+            return jit_compile(self.ccode, configuration['compiler'])
```

Comment:
This won't work with YASK. It needs to be ``self._compiler``. Perhaps this is post-merge/post-rebase leftover/error

Reply:
Indeed